src/Async.ts

```diff
@@ -18,7 +18,7 @@
  *   never started, no work is performed.)
  *
  * This API is curreid and has been optimized for use right-to-left
- * function composition like {@link pipe} or {@link flow}.
+ * function composition like pipe or flow.
```

Comment:
❓ would these `@link`s work if we imported pipe and flow at the top of this file?

Reply:
Got it working with Nick's help :grin: for reference: https://typedoc.org/guides/declaration-references/

https://github.com/fp-toolkit/fp-toolkit/pull/33/commits/25a048b0230f6983f9766431b286b35ee5ae10ac